Adding note about login slug to Auth0 (#3440)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/auth0-migration.md

SAML 2.0 users must update a few fields in the SSO app configuration to match the new Auth0 URL and URI.  You can approach this by editing the existing SSO app settings or creating a new one to accommodate the Auth0 settings. One approach isn't inherently better, so choose whichever works best for your organization.

:::warning Login {slug}

> The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins should share it with dbt Cloud users, as the login URL will be different.

Author: ```suggestion
The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins must share it with dbt Cloud users, as the login URL will differ.
```

SAML 2.0 users must update a few fields in the SSO app configuration to match the new Auth0 URL and URI.  You can approach this by editing the existing SSO app settings or creating a new one to accommodate the Auth0 settings. One approach isn't inherently better, so choose whichever works best for your organization.

:::warning Login {slug}

> Your login slug should not contain an underscore. If it does, it must be changed in the account settings.

Reviewer: changing to active voice

```suggestion
The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins should share it with dbt Cloud users, as the login URL will be different.
```

> Once the slug is changed, have your admins share it with dbt Cloud users, as the login URL will change.

Reviewer: ```suggestion
```
